### DISCUSSION TITLE
Disable indicators

### DISCUSSION THREAD

**Discussion by jjcortes22** [i3/i3#6101] [Q&A]
Good day,

I want to know if there is a way to remove the indicators completely, this is when moving windows with the mouse.

Thanks!

**Comment by okraits** [comment]
No, that's not possible afaik.

**Reply by jjcortes22** [reply]
Hi,

Thank you for the quick answer, can i make the color transparent somehow? when I drag and drop a window, i can change the color of the indicators but cannot make them transparent.
Gracias.

**Reply by okraits** [reply]
You could try and see if they are affected by a compositor.
